tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<ul><li>Link to folium one of the times it is mentioned (https://python-visualization.github.io/folium/)</li><li>Sp. visulatise &gt; visualize</li><li>zoom <strong>strat</strong> &gt; zoom <strong>level</strong></li><li>remove extra space: map : &gt; map:</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Done</p>